workorders.lic

```diff
@@ -28,7 +28,8 @@ class WorkOrders
     settings = get_settings
     crafting_data = get_data('crafting')
     @bag = settings.crafting_container('backpack')
-    info = crafting_data[discipline]
+    @bank_info = get_data('town')[settings.hometown]['deposit']['id']
+    info = crafting_data[discipline][settings.hometown]
```

Comment:
Will this work for Forging/Tailoring/Carving (i.e. the disciplines that aren't yet by-town)?